src/functions-and-arrays.js

```diff
@@ -52,14 +154,58 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+
+  findElements = (word, arr) => {
+    let indices = []
+    let element = word
+    let idx = arr.indexOf(element)
+    while (idx != -1) {
```

Comment:
comparações no javascript tente sempre utilizar 3 sinais de igualdade, ou `===` quando for pra ser igual, ou nesse caso `!==` se for diferente